src/components/accession2/viabilityTesting/ViewViabilityTestModal.tsx

```diff
@@ -70,7 +70,15 @@ export default function ViewViabilityTestModal(props: ViewViabilityTestModalProp
             </Grid>
             <Grid xs={12}>{selectedTest.seedType}</Grid>
           </Grid>
-          {!isMobile && (
+          {!isMobile && selectedTest.testType === 'Cut' && (
```

Comment:
rebasse? I had renamed the property to viabilityTest (since selectedTest semantic is specific to the parent component)